build-custom-fabric.sh

```diff
@@ -14,8 +17,8 @@ fi
 cd fabric
 
 # checkout a specific commit
-commit="8ce02b600c13"
 echo "Switching to commit=$commit"
+git checkout .
 git checkout $commit
```

Comment:
maybe better to make this explicitly `$tag` rather than commit, and amend this to `git checkout tags/$tag`

https://stackoverflow.com/questions/791959/download-a-specific-tag-with-git